### PR TITLE
System :n message

### PR DESCRIPTION
**Pull request by pbstFusion**
Changed the title to the thing in settings

### CODE REVIEW

**Review by Awesomewebm — CHANGES_REQUESTED** [review]
The Name of the Command Array `Notify = {` cannot have the same name, or they will override each other and mess it up.

**Comment by pbstFusion** [comment]
Riight

File: MainModule/Server/Core/Commands.lua [hunk 1]
```diff
@@ -1480,6 +1480,26 @@ return function(Vargs)
 				end
 			end
 		};
+		
+		
+		Notify = {
```
Comment (pbstFusion) [inline MainModule/Server/Core/Commands.lua:1485]: Change title to SystemNotify

**Comment by pbstFusion** [comment]
I CANNOT figure out how too change it

**Pull request closed without merging**